src/overlays/actors/ovl_En_Bb/z_en_bb.h

```diff
@@ -6,11 +6,56 @@
 
 struct EnBb;
 
+typedef void (*EnBbActionFunc)(struct EnBb*, GlobalContext*);
+
 typedef struct EnBb {
     /* 0x0000 */ Actor actor;
-    /* 0x014C */ char unk_14C[0x1DC];
+    /* 0x014C */ SkelAnime skelAnime;
+    /* 0x0190 */ Vec3s limbDrawTbl[16];
+    /* 0x01F0 */ Vec3s transitionDrawTbl[16];
+    /* 0x0250 */ s32 action;
+    /* 0x0254 */ s32 unk_254; //unused
+    /* 0x0258 */ EnBbActionFunc actionFunc;
+    /* 0x025C */ s32 moveMode;
+    /* 0x0260 */ s32 timer;
+    /* 0x0264 */ s32 unk_264; //unused
+    /* 0x0268 */ s16 actionState;
+    /* 0x026A */ s16 charge;
+    /* 0x026C */ s16 enBb_26C; // index for flame trail, phase of small hop when blue finds a bomb
+    /* 0x026E */ s16 enBb_26E; // move yaw for blue, flame timer for green, maxAlpha for flame trail
```

Comment:
think we talked about this in discord already, but the var doesnt need the index in it, and it should follow our camelcase scheme. some other generic name that covers all cases should be used (or just leave it unk if you dont want to figure it out)